app/components/remove_markdown/index.tsx

```diff
@@ -34,9 +34,9 @@ const RemoveMarkdown = ({enableEmoji, enableHardBreak, enableSoftBreak, textStyl
         return '\n';
     }, []);
 
-    const renderText = ({literal}: {literal: string}) => {
```

Comment:
Out of curiosity, why those changes in this file and what are the gains - did you measure?

Reply:
Not so much about gains, but behaviour, and a bit of self-serving.

Changing the style on the fly (that most of the time will happen only on development) was not updating the styles of renderText.

There are probably other minor performance improvements, since we are not creating anymore a new renderer on every render to be discarded, but I guess that was never a real problem.